content/docs/api-reference/make_checkpoint.md

```diff
@@ -52,42 +52,29 @@ Let's consider the following `dvc.yaml` file:
 
 ```yaml
 stages:
-  every100:
-    cmd: python iterate.py
+  train:
+    cmd: python train.py
     outs:
-      - int.txt:
+      - model:
           checkpoint: true
 ```
 
-The code in `iterate.py` will execute continuously increment an integer number
-saved in `int.txt` (starting at 0). At 0 and every 100 loops, it makes a
-checkpoint for [DVC experiments]:
+The code in `train.py` will train a model upto certain epochs. At every 100
+iterations, it saves the model to `model`, evaluates it and makes a checkpoint
+for [DVC experiments]:
 
 [dvc experiments]: /doc/user-guide/experiment-management#experiments
 
 ```py
-import os
-
 from dvc.api import make_checkpoint
```

Comment:
I wonder why we named it `make_checkpoint()` instead of just `checkpoint()`.

Reply:
I used `make_checkpoint` because `checkpoint` is a noun in this context and IMO method names should almost always be verbs.